doc/getting-started/building-from-source.rst

```diff
@@ -82,9 +82,10 @@ Install the necessary tools for the following systems:
           libnuma-dev \
           liblz4-tool \
           flex \
-          bison
+          bison \
+          xsltproc
 
-     $ sudo pip3 install kconfiglib
+     $ sudo pip3 install xmlschema
```

Comment:
During my testing, it's `lxml` that one needs to install, not `xmlschema`.

I'm actually not sure if `xmlschema` is needed at all, I've never installed it and could build all different configurations.

Reply:
Thanks for pointing this out. I've added `lxml` to the list.

About `xmlschema`, it is recommended to be installed as it allows further checking to the configuration XML files at configuration time. That said, we still allow building without it due to Yocto integration issues.